File: JavaArchitecture/app/src/main/java/com/demo/architecture/injection/component/FragmentComponent.java
package com.demo.architecture.injection.component;


import com.demo.architecture.injection.PerFragment;
import com.demo.architecture.injection.module.FragmentModule;

import dagger.Subcomponent;

/**
 * Created by HeYingXin on 2018/4/17.
 */
@PerFragment
@Subcomponent(modules = FragmentModule.class)
public interface FragmentComponent{

}

File: kotlin-frist-branch-TODO/app/src/main/java/com/mazouri/mvpkotlin/injection/component/FragmentComponent.kt
package com.mazouri.mvpkotlin.injection.component

import com.mazouri.mvpkotlin.injection.PerFragment
import com.mazouri.mvpkotlin.injection.module.FragmentModule
import dagger.Subcomponent

/**
 * Created by HeYingXin on 2018/4/17.
 */
@PerFragment
@Subcomponent(modules = arrayOf(FragmentModule::class))
interface FragmentComponent